src/components/FilterSearch.tsx

```diff
@@ -109,6 +109,11 @@ export function FilterSearch({
   const [currentFilter, setCurrentFilter] = useState<StaticFilter>();
   const [filterQuery, setFilterQuery] = useState<string>();
   const staticFilters = useSearchState(state => state.filters.static);
+  const matchingFilters: SelectableStaticFilter[] = useMemo(() => {
+    return staticFilters?.filter(({ filter, selected }) =>
+      selected && filter.kind === 'fieldValue' && searchFields.some(s => s.fieldApiName === filter.fieldId)
```

Comment:
nit can you put on their own line
```suggestion
      selected
      && filter.kind === 'fieldValue'
      && searchFields.some(s => s.fieldApiName === filter.fieldId)
```